cohorts/load.py

```diff
@@ -764,6 +767,43 @@ def _load_single_patient_effects(self, patient, only_nonsynonymous, filter_fn):
             patient=patient,
             filter_fn=filter_fn)
 
+    def load_kallisto(self):
+        """
+        Load a Kallisto gene expression data for a cohort
+
+        Parameters
```

Comment:
Is this a good practice to have an empty parameter list result in an empty docstring parameter list? (Not sure, just asking.)